src/OpenTelemetry.Exporter.Jaeger/ApacheThrift/Protocol/TProtocolException.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace Thrift.Protocol
 {
-    internal class TProtocolException : TException
+    internal sealed class TProtocolException : TException
```

Comment:
This should be automatically generated? How do we it in sync with the source?

Reply:
That's a good callout @reyang! I don't really have a solution. The Thrift stuff is highly customized. I documented here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Jaeger/ApacheThrift/README.md

It looks like the [latest ApacheThrift nuget](https://www.nuget.org/packages/ApacheThrift) has dropped the net45 ref so we might be able to take the official NuGet now if we bump JaegerExporter to .NET Framework 4.6.1 (I think that is the minimum to target netstandard2.0).

Reply:
open an issue to track this: https://github.com/open-telemetry/opentelemetry-dotnet/issues/2161

Reply:
@utpilla you can remove the changes from the auto-gen files.

Reply:
@cijothomas I have removed the changes from all the files under `src/OpenTelemetry.Exporter.Jaeger/ApacheThrift/`.